script.js

```diff
@@ -44,32 +41,108 @@ class ToDoList {
     }
 }
 
+// Currently we set up one default list, but it would be relatively straightforward
+// to add functionality to allow the user to create multiple lists.
 const toDoList = new ToDoList(true);
 
-// Joe: update display/DOM
 
-// Joe: update item in toDoList (to set it as "done")
+function updateDisplay(list) {
+
+    const outputElement = document.querySelector('section');
+    const items = list.getAllItems();
+    outputElement.innerHTML = '';
+
+    for (const item of items) {
+
+        const taskContainerEl = document.createElement('div');
 
-// (Stretch goal: hide completed items)
+        const toggleDoneInputEl = document.createElement('input');
+        toggleDoneInputEl.setAttribute('type', 'checkbox');
 
-/*
+        const taskTextContainerEl = document.createElement('span'); // change to div?
 
-    Add, update and delete functions *only* change the array.
-    (They only change the data that we're storing.)
+        // If the item's done flag is set to true:
+        if (item['done'] === true) {
+            // Wrap the text in a strikethrough tag:
+            taskTextContainerEl.innerHTML = `<s>${item['task']}</s>`;
+            // Check the checkbox:
+            toggleDoneInputEl.setAttribute('checked', 'on');
+            // Add the "done" class to this task's text container:
+            taskTextContainerEl.classList.add('done');
```

Comment:
Is this to add some CSS later?

Reply:
Yep! I'm wrapping "done" item text in the ``<s>`` tag for strikethrough, but it might be nice to have options to style the whole container (or any element within it), hence the class.